ash/src/extensions/khr/video_decode_queue.rs

```diff
@@ -0,0 +1,15 @@
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_video_decode_queue.html>
+
+use crate::vk;
+
+impl crate::khr::video_decode_queue::Device {
+    #[inline]
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdDecodeVideoKHR.html>
+    pub unsafe fn cmd_decode_video(
```

Comment:
Inline goes below the doc-comment, that makes it more uniform across the entire repo. And below `#[inline]`, I am trying to add `#[doc(alias = "vkCmdDecodeVideoKHR")]` too.

Reply:
You put the alias above the `inline` in the only example I found, but it should probably be below the docstring, no?

Reply:
"should" -> it "really doesn't matter", as long as there's one consistent ordering across the code-base to make (for example) `regex`es easier to work with.

It'd be great if `rustfmt` could deal with this for us.

Reply:
Adjusted the order to put the `#[doc(alias = "...")]` last.